templates/templates.go

```diff
@@ -0,0 +1,171 @@
+// Package templates contains variables holding embded template files.
+package templates
+
+import (
+	// Embed ...
```

Comment:
We can also help a future Nextmver understand this by saying something like this.
```suggestion
	// This package is required to embed files using the `//go:embed` directive albeit
	// it is not used directly.
```